packages/compass-aggregations/src/components/pipeline/pipeline.jsx

```diff
@@ -191,6 +193,47 @@ class Pipeline extends PureComponent {
     }
   }
 
+  renderPipelineToolbar() {
+    if (process.env.SHOW_NEW_AGGREGATION_TOOLBAR === 'true') {
+      return <PipelineToolbarLG />;
+    }
+    return (
+      <PipelineToolbar
```

Comment:
NIT: We could rename the old one to `<PipelineToolbarLegacy />` and have the new one already as `<PipelineToolbar>`.
Also we can flip the logic and keep the new one outside the flag, so that enabling the flag is just a matter of deleting code:

``` js
if (process.env.SHOW_NEW_AGGREGATION_TOOLBAR !== 'true') {
   return <LegacyPipelineToolbar ...>
}

return <PipelineToolbarLG />;
```